contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -702,23 +702,49 @@ NXmpes(NXobject):
         doc: |
           Calibrated energy axis.
           
-          This could be a link to either
-          /entry/process/energy_calibration/calibrated_axis or
-          /entry/process/energy_correction/calibrated_axis.
+          Could be linked from the respective '@reference' field.
         \@type:
+          type: NX_CHAR
+          doc: |
+            The energy can be either stored as kinetic or as binding energy.
+          enumeration:
+            kinetic:
+              doc:
+              - |
+                Calibrated kinetic energy axis.
+              - |
+                In case the kinetic energy axis is referenced to the Fermi level :math:`E_F`
+                (e.g., in entry/process/energy_referencing), kinetic energies :math:`E` are
+                provided as :math:`E-E_F`.
+              - |
+                xref:
+                  spec: ISO 18115-1:2023
+                  term: 3.35
+                  url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:3.35
+            binding:
+              doc:
+              - |
+                Calibrated binding energy axis.
+              - |
+                xref:
+                  spec: ISO 18115-1:2023
+                  term: 12.16
+                  url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.16
+        \@reference:
+          type: NX_CHAR
           exists: recommended
-      \@energy_depends:
-        type: NX_CHAR
-        exists: optional
-        doc: |
-          The energy can be dispersed according to different strategies. ``energy_depends`` points to
-          the path of a field defining the calibrated axis on which the energy axis depends.
-          
-          For example:
-            @energy_depends: 'entry/process/energy_calibration'
+          doc: |
+            The energy can be dispersed according to different strategies. ``@reference`` points to
+            the path of a field defining the calibrated axis which the ``energy`` axis refers.
+            
+            For example:
+              @reference: 'entry/process/energy_calibration/calibrated_axis'
+              @reference: 'entry/process/energy_calibration/calibrated_axis'
```

Comment:
Is that a duplicate?

Reply:
You are right, it was a copy past error. The second line was supposed to be `@reference: 'entry/process/energy_referencing/calibrated_axis'`.

I fixed it.